sdk/mobile-center-updates/src/androidTest/java/com/microsoft/azure/mobile/updates/BrowserUtilsTest.java

```diff
@@ -0,0 +1,328 @@
+package com.microsoft.azure.mobile.updates;
+
+import android.app.Activity;
+import android.content.ActivityNotFoundException;
+import android.content.Intent;
+import android.content.pm.ActivityInfo;
+import android.content.pm.PackageManager;
+import android.content.pm.ResolveInfo;
+import android.net.Uri;
+
+import org.junit.Test;
+import org.mockito.ArgumentMatcher;
+import org.mockito.InOrder;
+
+import java.util.Collections;
+
+import static com.microsoft.azure.mobile.updates.BrowserUtils.GOOGLE_CHROME_URL_SCHEME;
+import static java.util.Arrays.asList;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Matchers.argThat;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.inOrder;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+@SuppressWarnings("WrongConstant")
+public class BrowserUtilsTest {
```

Comment:
Since this is not urgent and test code, leave it as it is. We can revisit in the future.